Do not use -Werror, because the configure script incorrectly identifies clang
as gcc and uses warning flags clang does not understand.
https://sourceforge.net/p/autogen/bugs/185/
https://sourceforge.net/p/autogen/bugs/186/
--- configure.orig	2018-07-16 17:29:47.000000000 -0500
+++ configure	2018-07-23 21:26:03.000000000 -0500
@@ -18366,7 +18366,7 @@
 WARN_CFLAGS=
 test "X${GCC}" = Xyes && {
   CFLAGS="$CFLAGS -Wno-format-contains-nul -fno-strict-aliasing"
-  WARN_CFLAGS="$CFLAGS "`echo -Wall -Werror -Wcast-align -Wmissing-prototypes \
+  WARN_CFLAGS="$CFLAGS "`echo -Wall -Wcast-align -Wmissing-prototypes \
 	-Wpointer-arith -Wshadow -Wstrict-prototypes -Wwrite-strings \
 	-Wstrict-aliasing=3 -Wextra -Wno-cast-qual`
 }
